src/examples/Examples.js

```diff
@@ -51,10 +141,13 @@ var Examples = React.createClass({
               label: 'Examples',
               links: [{
                 label: 'Marquee',
-                href: '/docs/hpe/examples'
+                href: '/docs/hpe/develop/marquee'
```

Comment:
Originally there was a possibility we'd need multiple example pages with different templates/content, so I think that was the original intent of this menu.  I'm totally okay with changing it to be a menu for the documentation for each component, however if that's the case we should probably change the menu label from `Examples` to `Documentation`, or something like that.